recipes/libmaxminddb/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
+project(PackageTest C)
+
+find_package(PkgConfig)
+pkg_check_modules(LIBMAXMINDDB REQUIRED "libmaxminddb")
```

Comment:
I haven't seen a lot of examples using pkg_config in test_package, so having one more would be great.

Reply:
I'm happy to hear that. I love package config because the interface is lightweight and yet flexible to integrate with all (most?) build systems.